fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -80,7 +80,11 @@
       };
       reader.readAsArrayBuffer(update_file);
     } else {
-      fn(method, url, headers, null);
+      const turtle = document.getElementById('turtle_payload').value;
```

Comment:
Is the `turtle_payload` element available for _all_ views? If so, this is fine; otherwise, you may need

    const turtle = document.getElementById('turtle_payload')
    if (turtle && turtle.value) {
        headers.push(['Content-Type', 'text/turtle'])
        fn(method, url, headers, turtle.value)
    } else {
        fn(method, url, headers, null)
    }

For example, I wonder if this would be available on NonRDFSourceDescription pages, and also on Fixity check pages -- there may be others.